tests/bin/check_tron_jobs_test.py

```diff
@@ -65,3 +65,29 @@ def test_is_job_stuck_when_runtime_not_sorted(self):
         }
         run = check_tron_jobs.is_job_stuck(job_runs)
         assert_equal(run['id'], 'MASTER.test.2')
+
+    def test_no_job_scheduled_or_queuing(self):
```

Comment:
Doesn't have to block this review, but I think long-term we should have tests for compute_check_result_for_job_runs with various lists of job_runs and what we expect the check result to be, instead of testing the helpers.

Reply:
Good idea, I will add more tests later.